lib/sqlalchemy/dialects/postgresql/pg8000.py

```diff
@@ -409,7 +486,7 @@ def _dbapi_version(self):
 
     @classmethod
     def import_dbapi(cls):
-        return __import__("pg8000")
+        return __import__("pg8000.dbapi")
```

Comment:
this should be like this:

     return __import__("pg8000.dbapi").dbapi

python thing